llvm/lib/Target/RISCV/RISCVISelLowering.h

```diff
@@ -437,6 +436,9 @@ enum NodeType : unsigned {
   STRICT_FSETCCS_VL,
   STRICT_VFROUND_NOEXCEPT_VL,
   LAST_RISCV_STRICTFP_OPCODE = STRICT_VFROUND_NOEXCEPT_VL,
+  
+  INSERT_SUBVECTOR,
```

Comment:
TUPLE_INSERT and TUPLE_EXTRACT?

Reply:
This is a weird spot in the file to put these. Line 416, `STRICT_FCVT_W_RV64 = ISD::FIRST_TARGET_STRICTFP_OPCODE,` marked the start of opcodes that can cause FP exceptions. These opcodes should not be part of that range.

The VCIX opcodes are probably in the wrong place too.